src/implementations/twig/storybook/.storybook/preview-head.html

```diff
@@ -0,0 +1,39 @@
+<style>
```

Comment:
it loads the js, i added a small text about it in the pr description

Reply:
This file was used mainly for this 

```js
  if (
    inIframe() &&
    window.frameElement &&
    window.frameElement.id === 'storybook-preview-iframe'
  ) {
    // Let CSS Resources inject the styles
  } else {
    // Manually inject styles
    var head = document.head || document.getElementsByTagName('head')[0];
    head.appendChild(createLink('./styles/ec-core.css', 'screen'));
    head.appendChild(createLink('./styles/eu-core.css', 'screen'));
    head.appendChild(createLink('./styles/ec-core-print.css', 'print'));
    head.appendChild(createLink('./styles/eu-core-print.css', 'print'));
  }
```
Which will add styles in the fullscreen iframe `playground/ec/iframe.html?id=components-table--sortable&viewMode=story`  version of the story. I personally doubt anyone is using these fullscreen iframes and thus the deletion of the template.

Reply:
With regards to loading ECL and it's autoinit: ok, let's have the global loading back. The select multiple is the only component with non-default init, I'll find a way to destroy and re-init there.

Please remove the new addon and take the example preview-head from ecl-twig. I tried this and it worked

```
<script type="text/javascript" src="./scripts/ec-core.js"></script>
<script>
  // https://github.com/storybookjs/storybook/issues/6113#issuecomment-473965255
  function runOnPageChange() {
    ECL.autoInit();
  }

  document.addEventListener(
    'DOMContentLoaded',
    function () {
      const callback = function (mutationsList) {
        for (let i = 0, len = mutationsList.length; i < len; i++) {
          if (mutationsList[i].type == 'childList') {
            runOnPageChange();
            break;
          }
        }
      };
      const observer = new MutationObserver(callback);
      const config = { childList: true, subtree: false };
      observer.observe(document.getElementById('root'), config);
    },
    false
  );
</script>
```

If you want to take also the tippy for the compliance checker - ok for me.

The drawbacks of adding this new addon is loading scripts through a third party unnecessarily. When the init breaks, we'll be able to handle it on our side. I also don't see any advantage adding yet another new addon when we have a working solution in the ecl-twig

Reply:
I applied the same approach as in ecl-twig. Mind the fact that with this, the first page loaded will not have the autoInit called and we would need to call it also outside of this function.
Did not take all the scripts and styles for the compliance tool because in any case it's markup is based on utilities that do not exist yet in next-v3 so they would be pretty useless at the moment. We will reintroduce them once we will take care of this.

Reply:
What I meant in my previous comment was that the preview-head you had was ok, and you can only add the code after 
```
<script type="text/javascript" src="./scripts/ec-core.js"></script>
```
which is instantiating the ECL module. I did not mean that you should remove the css part, nor include the svg correction script. It's already here https://github.com/ec-europa/europa-component-library/blob/c546d79e2a4e3dbadfeb6a761b03ee73bb528870/src/implementations/twig/storybook/.storybook/ECL.jsx

Here's the contents of the file directly in order to avoid confusion

```html
<style>
  .sb-show-nopreview {
    display: none;
  }
</style>
<script>
  function inIframe() {
    try {
      return window.self !== window.top;
    } catch (e) {
      return false;
    }
  }

  function createLink(href, media) {
    var link = document.createElement('link');
    link.rel = 'stylesheet';
    link.type = 'text/css';
    link.href = href;
    link.media = media || 'all';
    return link;
  }

  if (
    inIframe() &&
    window.frameElement &&
    window.frameElement.id === 'storybook-preview-iframe'
  ) {
    // Let CSS Resources inject the styles
  } else {
    // Manually inject styles
    var head = document.head || document.getElementsByTagName('head')[0];
    head.appendChild(createLink('./styles/ec-core.css', 'screen'));
    head.appendChild(createLink('./styles/eu-core.css', 'screen'));
    head.appendChild(createLink('./styles/ec-core-print.css', 'print'));
    head.appendChild(createLink('./styles/eu-core-print.css', 'print'));
  }
</script>
<script type="text/javascript" src="./scripts/ec-core.js"></script>
<script>
  // https://github.com/storybookjs/storybook/issues/6113#issuecomment-473965255
  function runOnPageChange() {
    ECL.autoInit();
  }

  document.addEventListener(
    'DOMContentLoaded',
    function () {
      runOnPageChange();
      const callback = function (mutationsList) {
        for (let i = 0, len = mutationsList.length; i < len; i++) {
          if (mutationsList[i].type == 'childList') {
            runOnPageChange();
            break;
          }
        }
      };
      const observer = new MutationObserver(callback);
      const config = { childList: true, subtree: false };
      observer.observe(document.getElementById('root'), config);
    },
    false
  );
</script>
```

I've added `runOnPageChange();` invocation after dom is ready for running the ecl init on first run.